src/common/snippets/include/snippets/lowered/pass/brgemm_blocking.hpp

```diff
@@ -112,9 +113,68 @@ class BrgemmBlocking : public snippets::lowered::pass::RangedPass, public Brgemm
             size_t m_block, n_block, k_block;
             std::tie(m_block, n_block, k_block) = get_blocking_params(brgemm_expr);
             modified = mark_blocking_loops(linear_ir, expr_it, m_block, n_block, k_block);
+            SNIPPETS_DEBUG_PARAMS_DUMP(brgemm_expr, this);
         }
         return modified;
     }
+
+#ifdef SNIPPETS_DEBUG_CAPS
+    std::string params_to_csv(const ov::snippets::lowered::ExpressionPtr& brgemm_expr) {
+        const auto brgemm = ov::as_type_ptr<BRGEMM_TYPE>(brgemm_expr->get_node());
+        OPENVINO_ASSERT(brgemm, "Brgemm is nullptr!");
+        std::stringstream ss;
+        for (size_t i = 0; i < brgemm->get_input_size(); ++i) {
+            ss << brgemm->get_input_element_type(i);
+            if (i != brgemm->get_input_size() - 1) {
+                ss << ";";
+            }
+        }
+        ss << ",";
+        for (size_t i = 0; i < brgemm->get_output_size(); ++i) {
+            ss << brgemm->get_output_element_type(i);
+            if (i != brgemm->get_output_size() - 1) {
+                ss << ";";
+            }
+        }
+        ss << ",";
+        for (const auto& input : brgemm->inputs()) {
+            const auto& shape = input.get_partial_shape();
+            if (shape.is_dynamic()) {
```

Comment:
Please note: unless the whole shape can be dynamic, some specific dimension may still be static. So we need to check each dim separately. I'd also suggest converting the dim to `ov::Dimenstion`, which has `operator<<` overridden, to avoid custom printing logic with dynamic dim handling

Reply:
Thanks, done